src/components/summary/component.js

```diff
@@ -8,33 +8,47 @@ import Widget from 'components/widget';
 const Summary = ({ iso, categories }) => {
   const indicators = parseData(categories);
   return (
-    <ul className="c-summary">
-      {indicators.map(
-        ({ name, slug, indicatorsList, widgets }) =>
-          widgets &&
-          widgets.length > 0 && (
-            <li key={slug} className={classnames({ '-active': false })}>
-              <h2>{name}</h2>
-              <p>{indicatorsList}</p>
-              <div className="row">
-                {widgets.map(({ widgetSlug, widgetType }) => (
-                  <div key={widgetSlug} className="col-6">
-                    <Widget chart={widgetType} slug={widgetSlug} />
+    <section className="c-summary">
+      <div className="container">
+        {indicators.map(
+          ({ name, slug, indicatorsList, widgets }) =>
+            widgets &&
+            widgets.length > 0 && (
+              <article key={slug} className={classnames({ '-active': false })}>
+                <div className="row justify-content-md-center">
```

Comment:
In theory `container`, `row` and `col` shouldn’t have any other element among them. So in this case should be:

```
.container
  .row
     .col-12
        article
           .row
              .col-sm-12.col-md-10
```

Reply:
Yeah it sucks a bit, but it's the right way to use grid